support/k8s/k8s-workload-registrar/config_reconcile.go

```diff
@@ -159,16 +169,24 @@ func (slw SpiffeLogWrapper) Errorf(format string, args ...interface{}) {
 }
 
 // ServerID creates a server SPIFFE ID string given a trustDomain.
-func ServerID(trustDomain string) *spiretypes.SPIFFEID {
+func ServerID(trustDomain string) (*spiretypes.SPIFFEID, error) {
+	path, err := spiffeid.JoinPathSegments("spire", "server")
```

Comment:
why not to use `idutil.ServerIDPath` here?

Reply:
Good call.